### ISSUE TITLE
WordPress.WP.CronInterval.ChangeDetected can't handle intervals with parentheses

### ISSUE THREAD

**Issue by diedexx** [WordPress/WordPress-Coding-Standards#2025]
## Bug Description

The WordPress.WP.CronInterval.ChangeDetected sniff will show a `Detected changing of cron_schedules, but could not detect the interval value.` warning when the interval is surrounded with parentheses.

## Minimal Code Snippet
The issue happens when running this command:
```bash
phpcs ./myfile.php
```
... over a file containing this code:
``` php
// myfile.php
<?php

function add_cron_schedule( $schedules ) {
		$schedules['my-schedule'] = [
			'interval' => ( 15 * MINUTE_IN_SECONDS ), // <-- works without the "()"
			'display'  => esc_html__( 'Every fifteen minutes' ),
		];

		return $schedules;
}

add_filter( 'cron_schedules', 'add_cron_schedule' );
```

## Error Code

N/A

## Custom ruleset

I used the ruleset of this repository. 

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 7.2.22
| PHP_CodeSniffer version | 3.6.2
| WPCS version            | `develop` (2.3.0)
| WPCS install type       | Composer project local and git clone (haven't tested others)

## Additional Context (optional)

While debugging this I found that the parser does look for parentheses, but only parses the opening and not the closing.
After [this loop](https://github.com/WordPress/WordPress-Coding-Standards/blob/73f3d69b3a200935b80f5fd265bfbee8d642d082/WordPress/Sniffs/WP/CronIntervalSniff.php#L180) the `$value` is `"(15*MINUTE_IN_SECONDS"`

## Tested Against `develop` branch?
- [x] I have verified the issue still exists in the `develop` branch of WPCS.

**Comment by jrfnl** [comment]
@diedexx Thanks for reporting - see PR #2026

**Comment by diedexx** [comment]
Thank you @jrfnl for creating a patch so quickly!

**Comment by jrfnl** [comment]
Thanks for testing the patch @diedexx !
FYI: it may still be a while before the patch is released as it will go into WPCS 3.0.0...
